src/javascript/app_2/Stores/Modules/Contract/contract_store.js

```diff
@@ -49,15 +49,19 @@ export default class ContractStore extends BaseStore {
 
     @action.bound
     onUnmount = () => {
-        this.forgetProposalOpenContract();
+        this.smart_chart.removeBarriers();
+        this.smart_chart.removeMarkers();
+        // delay removal for animation transition
+        this.smart_chart.setContractMode(false);
+        setTimeout(() => this.removeContractInfo(), 600);
```

Comment:
This causes the next contract to become invalid. If the next contract initiated before this delay, then calling this afterwards will clear values.